### ISSUE TITLE
Set up nnUnet

### ISSUE THREAD

**Issue by bdzyubak** [bdzyubak/torch-control#1]
nnUnet is a state of the art image segmentation architecture that automatically configures hyperparameters to the GPU memory and dataset properties. I was able to obtain a major out-of-the-box organ segmentation accuracy compared to established Unet-based models at one of the companies I worked for. So, it makes sense to have nnUnet model available in the toolchain to compare to other state-of-the-art models within medical imaging, and in other fields. 

Paper: 
Isensee, F., Jaeger, P. F., Kohl, S. A., Petersen, J., & Maier-Hein, K. H. (2021). nnU-Net: a self-configuring method for deep learning-based biomedical image segmentation. Nature methods, 18(2), 203-211.

**Comment by bdzyubak** [comment]
nnUnet training curves (placing files here allows them to be linked in markdown docs): 
![image](https://github.com/bdzyubak/torch-control/assets/37943739/9f465a27-50f0-40a9-b43f-69914f3bc3cb)
![image](https://github.com/bdzyubak/torch-control/assets/37943739/149a92ff-d1fd-4f14-b31f-63df84017d26)
